### DISCUSSION TITLE
Can't renew SSL

### DISCUSSION THREAD

**Discussion by tokkoutai** [EasyEngine/easyengine#1559] [Support Requests] [answered]
Hi, I have this client trunning an old server with EE v3.7.5 and we tried to renew SSL and we are getting connection refused errors. Always worked fine so far. Googling it I saw I could try apt-get update and upgrade, did that and on update I get the following error:

`Err:10 http://download.opensuse.org/repositories/home:/rtCamp:/EasyEngine/xUbuntu_16.04  Release.gpg
  The following signatures were invalid: KEYEXPIRED 1571071577
Reading package lists... Done
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://download.opensuse.org/repositories/home:/rtCamp:/EasyEngine/xUbuntu_16.04  Release: The following signatures were invalid: KEYEXPIRED 1571071577
W: Failed to fetch http://download.opensuse.org/repositories/home:/rtCamp:/EasyEngine/xUbuntu_16.04/Release.gpg  The following signatures were invalid: KEYEXPIRED 1571071577`

I am guessing this version is no longer supported and the server should be update, problem is the client is having good traffic and do not want any down time for the time being. What could I do to at least make the ssl work? I do not understand much of it , could I do it manually somehow?

Thanks in advance

**Comment by tokkoutai** [comment, marked as answer]
For anyone else getting this, I simply done it manually with certbot. Followed these tutorials:

https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-20-04-pt
https://www.penta-code.com/how-to-get-free-https-in-10-minutes-with-letsencrypt-and-certbot/

**Reply by kirtangajjar** [reply]
@tokkoutai EasyEngine 3.x is no longet supported. You can migrate the site to a popular fork of EE3 - https://wordops.net/
